webapp/Components/HeroAreaComponent.razor

```diff
@@ -0,0 +1,54 @@
+                            <h2 class="wow fadeInUp" data-wow-delay=".3s">Welcome to ClassiGrids</h2>
+                            <p class="wow fadeInUp" data-wow-delay=".5s">Buy And Sell Everything From Used Cars To Mobile Phones And <br>Computers,
+                                Or Search For Property, Jobs And More.</p>
+                        </div>
+                        <!-- End Search Form -->
+                        <!-- Start Search Form -->
+                        <div class="search-form wow fadeInUp" data-wow-delay=".7s">
+                            <div class="row d-flex justify-content-center">
+                                <div class="col-lg-4 col-md-4 col-12 p-0">
+                                    <div class="search-input">
+                                        <label for="keyword"><i class="lni lni-search-alt theme-color"></i></label>
+                                        <input type="text" name="keyword" id="keyword" placeholder="Product keyword">
+                                    </div>
+                                </div>
+                                <div class="col-lg-3 col-md-3 col-12 p-0">
+                                    <div class="search-input">
+                                        <label for="category"><i class="lni lni-grid-alt theme-color"></i></label>
+                                        <select name="category" id="category">
+                                            <option value="none" selected disabled>Categories</option>
+                                            <option value="none">Vehicle</option>
+                                            <option value="none">Electronics</option>
+                                            <option value="none">Mobiles</option>
+                                            <option value="none">Furniture</option>
+                                            <option value="none">Fashion</option>
+                                            <option value="none">Jobs</option>
+                                            <option value="none">Real Estate</option>
+                                            <option value="none">Animals</option>
+                                            <option value="none">Education</option>
+                                            <option value="none">Matrimony</option>
+                                        </select>
+                                    </div>
+                                </div>
+                                <div class="col-lg-2 col-md-2 col-12 p-0">
+                                    <div class="search-btn button">
+                                        <button class="btn"><i class="lni lni-search-alt"></i> Search</button>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                        <!-- End Search Form -->
```

Comment:
@Abubakr0904 This must be a separate component

Reply:
@wahid-d shundemi?